scripts/build/ci-build/Dockerfile

```diff
@@ -113,8 +113,8 @@ FROM debian:stretch-20210208
 ENV GOVERSION=1.17 \
     PATH=/usr/local/go/bin:$PATH \
     GOPATH=/go \
-    NODEVERSION=14.17.6-1nodesource1 \
-    YARNVERSION=3.1.0-rc.5
+    NODEVERSION=14.18.1-1nodesource1 \
+    YARNVERSION=1.22.15-1
```

Comment:
Should this be
```suggestion
    YARNVERSION=3.1.0
```
?

Reply:
Thought so too, but I get the error that this version doesn't exist.
Actually, if you don't specify a version, the latest version it gets is this one. (`1.22.15-1`)

Reply:
If this Drone run passes, then I think we are okay to go :)